tools/tests/test_maskfile_configuration.py

```
@@ -0,0 +1,236 @@
+        assert "describe-images" in content or "batch-get-image" in content or "content_hash" in content, \
+            "Push command must check if image already exists before pushing"
+
+    def test_has_skip_push_logic(self) -> None:
+        content = _read_maskfile()
+        assert "skipping push" in content.lower() or "skip" in content.lower(), \
+            "Must have logic to skip push when image is unchanged"
+
+    def test_pushes_with_commit_tag(self) -> None:
+        content = _read_maskfile()
+        assert "git" in content.lower() and ("tag" in content.lower() or "rev-parse" in content.lower()), \
+            "Should tag images with git commit for traceability"
```

Ruff (0.14.14)

[warning] Assertion should be broken down into multiple parts

(PT018)